src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -132,6 +129,7 @@ public boolean equals(Object other) {
         private Name name;
         private Phone phone;
         private Email email;
+        private DateTime dateTime;
```

Comment:
the dateTime class is supposed to be the super class for datetime fields in the model.